packages/faucet/src/database-helper.ts

```diff
@@ -26,11 +26,19 @@ export enum RequestType {
   Invite = 'Invite',
 }
 
+enum MobileOS {
+  'andriod' = 'android',
```

Comment:
typo `andriod`. Also, why the `''` in the key name?

Reply:
no good reason just moving fast